docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -195,6 +200,11 @@ and should appear in the `[BlockStorage]` section of the `cloud.conf` file:
   provided by Cinder. The default value of `false` results in the discovery of
   the device path based on it's serial number and /dev/disk/by-id mapping and is
   the recommended approach.
+* `ignore-volume-az` (Optional): Used to influence availability zone use when
+  attaching Cinder volumes. When Nova and Cinder have different availability
+  zones, this should be set to `true`. This is most commonly the case where
+  there are many Nova availability zones but only one Cinder availability zone.
+  The default value is `false`.
```

Comment:
may be we should say that the default value is false to preserve compatibility with earlier versions and we may revisit the default in the future?

Reply:
Done!